masterlist.yaml

```diff
@@ -478,6 +478,23 @@ globals:
       - '[New Vegas Script Extender](http://nvse.silverlock.org/)'
     condition: 'file("NVSE/Plugins/([^\.]+\.dll)") and not file("../nvse_1_\d(ng)?\.dll")'
 
+# NVSR
+  - <<: *supersede
+    subs:
+      - '[New Vegas Stutter Remover](https://www.nexusmods.com/newvegas/mods/34832/)'
+      - '[NVTF - New Vegas Tick Fix](https://www.nexusmods.com/newvegas/mods/66537/)'
+    condition: 'file("NVSE/Plugins/sr_New_Vegas_Stutter_Remover.dll")'
```

Comment:
It would be good to expand the condition like so:
```yaml
    condition: 'file("NVSE/Plugins/sr_New_Vegas_Stutter_Remover.dll") and not file("NVSE/Plugins/NVTF.dll")'
```